Lib/test/test_code.py

```diff
@@ -161,6 +168,7 @@ def dump(co):
                  "kwonlyargcount", "names", "varnames",
                  "cellvars", "freevars", "nlocals", "flags"]:
         print("%s: %s" % (attr, getattr(co, "co_" + attr)))
+    print("lnotab:", repr(co.co_lnotab))
```

Comment:
Minor: maybe something like this would result in more readable output?

```suggestion
    print("lnotab:", list(co.co_lnotab))
```